assets/patterns/messages/messages-read.html

```diff
@@ -0,0 +1,23 @@
+<table id="messagesTable" class="messages-table">
```

Comment:
I don't think this should be a `table` as it's not tabular data. It also won't collapse well on smaller viewports.